### PR TITLE
Add recipe for matrix-py3

### PR DESCRIPTION
**Pull request by chunribu**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by chunribu** [comment]
@conda-forge/help-python  Ready for review.

File: recipes/matrix-py3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  skip: True  # [py<3 or win]
+  script: "{{ PYTHON }} -m pip install ."
+
+requirements:
+  build:
```
Comment (jaimergp) [inline recipes/matrix-py3/meta.yaml:39]:
```suggestion
  host:
```

File: recipes/matrix-py3/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  skip: True  # [py<3 or win]
```
Comment (jaimergp) [inline recipes/matrix-py3/meta.yaml:35]:
```suggestion
  skip: True  # [win]
```

File: recipes/matrix-py3/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (jaimergp) [inline recipes/matrix-py3/meta.yaml:1]: Can you remove the comments please?

File: recipes/matrix-py3/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "matrix-py3" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/54/c8/ef3647906d37e41fbda34a6ffbe8bfc5633dd44276be2ff7844ee52b4ae9/matrix-py3-0.0.1.tar.gz
```
Comment (jaimergp) [inline recipes/matrix-py3/meta.yaml:14]: The form mentioned [in the example](https://github.com/chunribu/staged-recipes/blob/master/recipes/example/meta.yaml#L14) is preferred for PyPI packages:

File: recipes/matrix-py3/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  run:
+    - python
+    - ncurses
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - matrix3
+  # For python packages, it is useful to run pip check. However, sometimes the
```
Comment (jaimergp) [inline recipes/matrix-py3/meta.yaml:53]:
```suggestion
  requires:
    - pip
  commands:
    - pip check
  # For python packages, it is useful to run pip check. However, sometimes the
```

**Comment by chunribu** [comment]
@jaimergp Thank you very much for your detailed guidance! It looks more standard now.

**Comment by jaimergp** [comment]
Thanks!

**Pull request merged**